src/components/js/EcApplication.js

```diff
@@ -0,0 +1,86 @@
+import VueMarkdown from 'vue-markdown'
+import EcomApps from '@ecomplus/apps-manager'
+
+export default {
+  name: 'EcApplication',
+  components: {
+    VueMarkdown
+  },
+  data () {
+    return {
+      adminSettings: {},
+      showSettings: false,
+      loading: false,
+      data: null,
+      hiddenData: null
+    }
+  },
+  props: {
+    ecomApps: {
+      type: Object,
+      default: () => new EcomApps()
+    },
+    application: {
+      type: Object,
+      default: () => { }
+    },
+    localApplication: {
```

Comment:
`application` e `localApplication` como prop? Não faz sentido.
O `localApplication` era pra ser do data ou computed, assim você pode alterá-lo diretamente, se for só pra emitir o evento `update` (e não alterar diretamente) você não precisa do `localApplication`

Reply:
Na verdade eu to tratando o application como o aplicativo que é feito fetch do market, e o localApplication como o application que vem da store api. Mas comecei a entender melhor as coisas com o artigo que voce deixou ai acima.